services/nagprainventory/jaxb/src/main/resources/nagprainventory_common.xsd

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+
+<!--
+    NagpraInventory schema (XSD)
+
+    Entity  : NagpraInventory
+    Part    : Common
+    Used for: JAXB binding between XML and Java objects
+-->
+
+<xs:schema
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:jaxb="http://java.sun.com/xml/ns/jaxb"
+  jaxb:version="1.0" elementFormDefault="unqualified"
+  xmlns:ns="http://collectionspace.org/services/nagprainventory"
+  xmlns="http://collectionspace.org/services/nagprainventory"
+  targetNamespace="http://collectionspace.org/services/nagprainventory"
+  version="0.1"
+>
+
+<!--
+    Avoid XmlRootElement nightmare:
+    See http://weblogs.java.net/blog/kohsuke/archive/2006/03/why_does_jaxb_p.html
+-->
+
+  <!--  NagpraInventory Information Group -->
+  <xs:element name="nagprainventories_common">
+    <xs:annotation>
+      <xs:appinfo>
+        <jaxb:class name="NagpraInventoriesCommon" />
+      </xs:appinfo>
+    </xs:annotation>
```

Comment:
@ray-lee I wanted to highlight this as something new I was trying and is different from the other procedures. It just helps to generate a class name that's more consistent with what we'd expect, while still keeping the `napgrainventories_common` namespace in tact.